sys-libs/mesa/mesa-22.0.5.recipe

```diff
@@ -122,6 +122,7 @@ BUILD()
 		-Dgallium-drivers=swrast \
 		-Dvulkan-drivers=swrast \
 		-Dvulkan-icd-dir=$addOnsDir/vulkan/icd.d \
+		-Dshader-cache=false \
```

Comment:
Why add this?

Reply:
The Intel driver does not interact with MESA in any way. Please remove this flag.

Reply:
Use MESA_SHADER_CACHE_DISABLE=1 if any app has an issue with it.